blogs/13_bfcl_v3_multi_turn.html

```diff
@@ -743,73 +743,62 @@ <h4>5. API Code Validation (🧑‍💻+ 💻 )</h4>
-            evolves. Each function call alters the state, such as creating new files, changing configurations, or
-            updating logs. At every turn, we compare the instance’s current state from executing the model function
-            calls with the expected ground truth state from executing the ground truth function calls to determine if
-            the sequence of function calls is correct.</p>
-          <p>If any discrepancy is detected between the model’s instance state and the expected state for the ground
-            truth instance, the evaluation flags the issue as a failure for that turn.</p>
-
-          <h4>4. Why We Avoid Certain Techniques (e.g. ReAct)</h4>
+          <h4>Why We Avoid Certain Techniques (e.g. ReAct)</h4>
           <p>In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, we deliberately avoid using techniques like prompt engineering and ReAct, which combines
             reasoning and acting through specific prompting methods. While ReAct and other techniques can improve
-            models’ function calling performance in certain cases, we chose not to use it throughout the BFCL series
+            models' function calling performance in certain cases, we chose not to use it throughout the BFCL series
             to evaluate base LLMs with the same standards to isolate the effects from using additional optimization
             techniques.</p>
-
         </div>
 
         <div>
-          <h3 id="evaluation">Multi-turn Evaluation Metrics (State-based Evaluation)</h3>
-          <p>In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, state-based evaluation is the primary metric used to assess the performance of models in
-            multi-turn function-calling scenarios. This approach focuses on comparing the instance’s final state after
-            all function calls are executed at each turn of the conversation. The key idea is to track how the
-            system's internal state changes after each step in the interaction and ensure that it aligns with the
-            expected state trajectory.</p>
-          <p>Response-based evaluation is an alternative approach, which evaluates 1) the function calling trajectory
-            and 2) intermediate execution response equivalence of the function calls in each turn. Previous versions,
-            BFCL V1 and V2 • Live, used Abstract Syntax Tree (AST) and Executable categories for this method. In the
-            following sections, we discuss the advantages of state-based evaluation and some limitations of
-            response-based evaluation in multi-turn function calling.</p>
-
-          <h4>Why State-based Evaluation is More Effective</h4>
+          <h3 id="evaluation">Multi-turn Evaluation Metrics</h3>
+          <p>In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, both state-based evaluation and response-based evaluation are used to assess the model's performance the multi-turn categories. </p>
```

Comment:
bold `both state-based evaluation and response-based evaluation are used to assess the model's performance the multi-turn categories`

Reply:
Maybe add a small visualization that's doing AND operation on the output of three checkers (state, response, not force-terminated) will help for understanding.